test/transforms.jl

```diff
@@ -680,6 +680,49 @@
     @test n1 == n2
   end
 
+  @testset "StdNames" begin
+    # camel, snake, upper tests
+    @test TableTransforms._camel("apple banana") == "AppleBanana"
+    @test TableTransforms._snake("apple banana") == "apple_banana"
+    @test TableTransforms._upper("apple banana") == "APPLEBANANA"
+
+    @test TableTransforms._camel("apple_banana") == "Apple_banana"
+    @test TableTransforms._snake("apple_banana") == "apple_banana"
+    @test TableTransforms._upper("apple_banana") == "APPLE_BANANA"
+
+    @test TableTransforms._camel("apple_Banana") == "Apple_Banana"
+    @test TableTransforms._snake("apple_Banana") == "apple_banana"
+    @test TableTransforms._upper("apple_Banana") == "APPLE_BANANA"
+
+    @test TableTransforms._camel("a") == "A"
+    @test TableTransforms._snake("a") == "a"
+    @test TableTransforms._upper("a") == "A"
+
+    @test TableTransforms._camel("B") == "B"
+    @test TableTransforms._snake("B") == "b"
+    @test TableTransforms._upper("B") == "B"
+
+    # row table test
+    names = (:a, Symbol("apple tree"), Symbol("banana tree"))
+    cols = ([1,2,3], [4,5,6], [7,8,9])
+    t = Table(; zip(names, cols)...)
+
```

Comment:
Why there is an empty line here? Can you please try to be more consistent with the way you group lines of code? It helps a lot the reviewer understand your intention.